com.unity.render-pipelines.high-definition/Runtime/Lighting/LightEvaluation.hlsl

```diff
@@ -472,6 +471,50 @@ float EvaluateShadow_Punctual(LightLoopContext lightLoopContext, PositionInputs
 #endif
 }
 
+
+SHADOW_TYPE EvaluateShadow_RectArea( LightLoopContext lightLoopContext, PositionInputs posInput,
+                                     LightData light, BuiltinData builtinData, float3 N, float3 L, float dist)
+{
+#ifndef LIGHT_EVALUATION_NO_SHADOWS
+    SHADOW_TYPE shadow  = 1.0;
+    float shadowMask    = 1.0;
+    float NdotL         = dot(N, L); // Disable contact shadow and shadow mask when facing away from light (i.e transmission)
+
+#ifdef SHADOWS_SHADOWMASK
+    // shadowMaskSelector.x is -1 if there is no shadow mask
+    // Note that we override shadow value (in case we don't have any dynamic shadow)
+    shadow = shadowMask = (light.shadowMaskSelector.x >= 0.0 && NdotL > 0.0) ? dot(BUILTIN_DATA_SHADOW_MASK, light.shadowMaskSelector) : 1.0;
+#endif
+
+#if defined(SCREEN_SPACE_SHADOWS) && !defined(_SURFACE_TYPE_TRANSPARENT) && (SHADERPASS != SHADERPASS_VOLUMETRIC_LIGHTING)
+    if ((light.screenSpaceShadowIndex & SCREEN_SPACE_SHADOW_INDEX_MASK) != INVALID_SCREEN_SPACE_SHADOW)
+    {
+        shadow = GetScreenSpaceShadow(posInput, light.screenSpaceShadowIndex);
+    }
+    else
+#endif
+    if ((light.shadowIndex >= 0) && (light.shadowDimmer > 0))
+    {
+        // light.positionRWS now contains the Light vector.
```

Comment:
This comment seems out of place? light.positionRWS is not referenced.

Reply:
indeed, come from the original code, I will removed

Reply:
fixed